x-pack/plugin/security/src/test/java/org/elasticsearch/xpack/security/authc/AuthenticationServiceTests.java

```diff
@@ -567,24 +582,23 @@ public void testTokenRestMissing() throws Exception {
         });
     }
 
-    public void authenticationInContextAndHeader() throws Exception {
+    public void testAuthenticationInContextAndHeader() throws Exception {
```

Comment:
This test was skipped for a long time because of the naming convention mismatch. I fixed but some of it's not 100% the same as it was because things evolved.